### ISSUE TITLE
[Feature Request] 

### ISSUE THREAD

**Issue by al-jimenez** [benphelps/homepage#245]
Currently getting widget status indicators requires you to click/select the green status indicator to the right of a service
Great hidden feature as I was not aware of it.  

I think a neat feature, since this is a great docker dashboard, would be to have a button at the bottom (just like green status indicator) (that only appears when a docker service widget is defined),  that when clicked will act as a global status enable for all docker services listed.

In other words, if you click the green dot at the bottom right of page, all docker service containers will expand their status indictors.    

This would be a great addition to this great dashboard.  Good job !

**Comment by henryxrl** [comment]
Or, for each docker container, add the option to expand by default, like ```expends: true```

**Comment by JazzFisch** [comment]
As Ben explained in this discussion [Expand docker container resouces stats by default.](https://github.com/benphelps/homepage/discussions/390#discussioncomment-3887770) this presents performance problems.

> It could be possible, but there are problems that arise when doing this. The docker stats take ~1.5 seconds to load, as the CPU needs to be sampled to gather the usage percentage. Expanding the stats for 6 resources effectively causes the page to take 3-4 seconds extra to load. This compounds with the more you have expanded. By default the browser only makes 3 concurrent requests, so once it's making 3 docker stats requests, everything else gets delayed.
